engines/private/private.cpp

```diff
@@ -467,6 +495,74 @@ bool PrivateEngine::cursorPauseMovie(Common::Point mousePos) {
 	return false;
 }
 
+Common::String PrivateEngine::getPauseMovieSetting() {
+	if (_language == "us")
```

Comment:
Right here, if you switch `_language` to `Common::Language`, then `if (_language == Common::EN_USA)`

Reply:
Done.